.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: elixir
 elixir:
   - 1.4.1
+cache:
+  directories:
+    - _build
```

Comment:
ok I'll create an issue link to this PR, thanks for creating the PR :+1: and feel free to add more info or if you are sure that the build folder can be cached let's reopen the PR